test/markup/javascript/keywords.expect.txt

```diff
@@ -1,6 +1,6 @@
 <span class="hljs-function"><span class="hljs-keyword">function</span> <span class="hljs-title">$initHighlight</span>(<span class="hljs-params">block, cls</span>) </span>{
   <span class="hljs-keyword">try</span> {
-    <span class="hljs-keyword">if</span> (cls.search(<span class="hljs-regexp">/\bno\-highlight\b/</span>) != <span class="hljs-number">-1</span>)
+    <span class="hljs-keyword">if</span> (cls.search(<span class="hljs-regexp">/\bno\-highlight\b/</span>) != -<span class="hljs-number">1</span>)
```

Comment:
That looks like a regression, but maybe it's fine.

Reply:
```
<span class="hljs-keyword">let</span> diff = <span class="hljs-number">5</span><span class="hljs-number">-3</span>;
```

![Screen Shot 2020-10-03 at 10 44 05 AM](https://user-images.githubusercontent.com/6473/94994401-62104000-0565-11eb-9812-0276efc3aebf.png)

I'd say actually it's a FIX - just we didn't have good numeric tests in the past. :-)  C_NUMBER includes `-?` in the regex, which as I said is problematic... it's probably something we should consider changing (for exactly this reason), but that's a separate issue.

When look behind is added we could add `-` back when it immediately follows certain things like `(` or `=`, but that's also another day.